components/app/PlaceItem.jsx

```diff
@@ -5,11 +5,12 @@ import { isLast } from '../utils';
 
 import { Box, Text, Image } from '../atoms';
 import { Row } from '../molecules';
-
+import { Link } from "gatsby";
+import _ from "lodash";
 
 
 const PlaceImage = ({ source, ...props }) => (
-  <Image source={source} {...props} />
+  <Image source={"http://drive.google.com/uc?export=view&id=" + source.split("=")[1]} {...props} />
```

Comment:
This kind of logic should exist in the container component (`PlaceItemContainer`).

I would probably do something like `if (source.startsWith('drive:')) { src = source.replace('drive:', 'http://drive.google.com/uc?export=view&id=')...`, to make sure normal URLs still work

Reply:
I'm not really sure how to do this!  I've had a go but am a bit clueless where it goes in `(PlaceItemContainer)`.  Do you mind adding it sorry!

Reply:
Added this to 55f8459 .